lib/attribute.js

```diff
@@ -455,7 +456,10 @@ validators.format = function validateFormat (instance, schema, options, ctx) {
   if (typeof instance === 'undefined') {
```

Comment:
We should also fix all the other `undefined` references in here to handle `null` as well